### ISSUE TITLE
Creating temporary table crashes pgcli 0.14.0

### ISSUE THREAD

**Issue by samgre** [amjith/pgcli#157]
Hi,

Thanks for a great command line tool! Try running the following SQL statement in the interpreter. It'll crash the app.

    CREATE TEMPORARY TABLE tbl (col INT);

The error:

    File "[...]/lib/python2.7/site-packages/pgcli/pgcompleter.py", line 99, in extend_tables
        self.dbmetadata[schema][table] = ['*']
    KeyError: u'pg_temp_4'

**Comment by amjith** [comment]
Thanks for reporting this issue. 

I think @darikg fixed this bug recently (unreleased) #151. 

Would you mind trying the version from master? 

You can install directly from master using the following command

```
pip install -U -e git+https://github.com/amjith/pgcli.git#egg=pgcli
```

**Comment by samgre** [comment]
Creating temporary table works with the current version of master. Thank you!
